src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
```

Comment:
Any additional latency in a time code set-up will make the things worth. The user monitors the result by headphones and what he does should be almost instantly heard. If he hears no double beats in his headphones, there should be non on the dance-floor. This is by the way no difference if he users a turntable or a midi controller to cue the track in.